aws/resource_aws_batch_job_queue.go

```diff
@@ -19,6 +19,13 @@ func resourceAwsBatchJobQueue() *schema.Resource {
 		Update: resourceAwsBatchJobQueueUpdate,
 		Delete: resourceAwsBatchJobQueueDelete,
 
+		Importer: &schema.ResourceImporter{
```

Comment:
Please note: This pull request currently contains the changes from #11406, which need adjustments. If the `aws_batch_job_queue` changes are removed from this pull request, we can likely get the `aws_batch_job_definition` changes (which look okay at first glance) merged sooner.